src/modules/map/components/openlayers/OpenLayersWMSLayer.vue

```diff
@@ -51,35 +53,75 @@ export default {
         },
     },
     created() {
-        let tileGrid = undefined
+        let source = undefined
+        if (this.gutter !== -1) {
+            source = new TileWMS({
+                projection: this.projection,
+                url: this.url,
+                gutter: this.gutter,
+            })
+        } else {
+            source = new ImageWMS({
+                url: this.url,
+                projection: this.projection,
+            })
+        }
         // If we are using LV95, we can constrain the WMS to only request tiles over Switzerland
         if (this.projection === CoordinateSystems.LV95.epsg) {
-            tileGrid = new TileGrid({
+            const tileGridLV95 = new TileGrid({
                 resolutions: TILEGRID_RESOLUTIONS,
                 extent: TILEGRID_EXTENT,
                 origin: TILEGRID_ORIGIN,
                 tileSize: WMS_TILE_SIZE,
             })
+
+            if (this.gutter !== -1) {
+                source = new TileWMS({
+                    projection: this.projection,
+                    url: this.url,
+                    gutter: this.gutter,
+                    tileGrid: tileGridLV95,
+                })
+            } else {
+                source = new ImageWMS({
+                    projection: this.projection,
+                    url: this.url,
+                    tileGrid: tileGridLV95,
+                })
+            }
```

Comment:
This looks really the same as line 57 to 68, would it be possible to use some setter for the TileGrid instead of recreating the same instances with this new option?